app/assets/stylesheets/search.scss

```diff
@@ -90,23 +97,55 @@
 }
 
 .my-flight {
-  border-bottom: 1px solid $my-gray;
+  border: 1px solid $light-gray;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
   font-size: 0.85em;
-  height: 4em;
-  padding: 0;
+  vertical-align: center;
+  p{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space